austrakka/utils/options.py

```diff
@@ -234,6 +234,7 @@ def opt_project(**attrs: t.Any):
         'help': 'Project Abbreviation',
     }
     return _create_option(
+        '-p',
```

Comment:
This is currently dangerous as the `--proforma` option has the same `-p` short form, and click won't handle this intelligently - if we ever add both options to the same command, they will clash